Java/Matrix/Problems.java

```diff
@@ -36,4 +36,7 @@ public static void main(String[] args) {
         p.searchInSortedMatrix(matrix, 16);
         
     }
+
```

Comment:
have you made any significant changes in this file or just pushed this file accidently?
If by accident, I will remove this commit.

Reply:
It was an accident